interoperability-tests/Testing-between-Gramine-and-IntelSGXSDK.md

```diff
@@ -0,0 +1,147 @@
+Testing between Gramine and Intel SGX SDK RA-TLS
+====
+
+# Introduction
+
+This document records the interoperability testing between [Gramine](https://github.com/gramineproject/gramine) and [Intel SGX SDK](https://github.com/intel/linux-sgx), both of which have implemented Interoperable RA-TLS.
+
+We conducted two sets of tests, in which Gramine and Intel SGX SDK RA-TLS respectively verified each other's certificates and established TLS secure channels. You can verify our tests by following the steps below.
+
+# Prerequisites
+
+## Intel SGX SDK and PSW packages
+
+To run the tests, first you need to download and install Intel SGX SDK and corresponding PSW packages on your system. 
+
+Interoperable RA-TLS was implemented since [Intel SGX SDK 2.22](https://github.com/intel/linux-sgx/releases/tag/sgx_2.22) 
+
+Get Intel SGX SW packages from [Intel SGX SW latest](https://download.01.org/intel-sgx/latest/linux-latest/distro/), and install the Intel SGX SDK and PSW packages by following [Intel SGX SW Installation Guide](https://download.01.org/intel-sgx/latest/linux-latest/docs/Intel_SGX_SW_Installation_Guide_for_Linux.pdf).
+
+# Intel SGX SDK RA-TLS as attester and Gramine as verifier
+
+## Intel SGX SDK RA-TLS
+
+Intel SGX SDK RA-TLS provides a sample program [SampleAttestedTLS](https://github.com/intel/linux-sgx/tree/master/SampleCode/SampleAttestedTLS), which can be used to start an RA-TLS server.
+
+
+1. Get source code of SampleAttestedTLS
+
+```sh
+git clone https://github.com/intel/linux-sgx
+```
+
+2. Build the sample
+
+```sh
+cd SampleCode/SampleAttestedTLS
+
```

Comment:
Please remove this empty line between `cd` and `make`

Reply:
updated.